rios/calcstats.py

```diff
@@ -337,11 +337,20 @@ def calcStats(ds,progress=None,ignore=None,
     """
     if progress is None:
         progress = cuiprogress.SilentProgress()
+    
+    if ignore is not None:
+        setNullValue(ds, ignore)
```

Comment:
Should we remove the `SetNoData` call from `addStatistics`? Or leave it in there?

Reply:
I thought about this, but decided not to, in case there is application code out there which directly calls addStatistics. 

Also, this is the reason for my comment saying "also set inside addStatistics" - I am planning on leaving that in place, and wanted to clarify that it is also called inside that routine, which is redundant in the context of the close() method.

Separately, the null value is set by calcStats, but it turns out that calcStats is never called from within Rios, and exists only for use by applications.

Reply:
Ahh right - fair enough